agreements/project-contribution/asset-transfer-agreement.md

```diff
@@ -65,17 +65,17 @@ The attached [schedules of assets](#schedules) identify the assets being transfe
 
 **Software**:  The computer programs associated with the Project, and their documentation, over which the Assignor exercises control through hosting or acting as maintainer.
 
-**Source Code Repositories**: The version control repositories used by the Project to maintain its software, documentation, and related artifacts, over which the Assignor exercises control through hosting or acting as maintainer.
+**Source Code Repositories**: The repository accounts (such as GitHub organizations, repositories, or bots) used by the Project to maintain its software, documentation, and related materials, and currently controlled by the Assignor as host or maintainer. For the avoidance of doubt, this does not include any code or other content maintained in the repositories.
```

Comment:
Is ` For the avoidance of doubt, this does not include any code or other content maintained in the repositories` because that's covered under the *Software* definition

Reply:
Specifically requested for clarity and continued avoidance of doubt.